katcp/test/test_resource_client.py

```diff
@@ -1245,7 +1271,8 @@ def setUp(self):
             'resource1' : dict(controlled=True),
             'resource2' : dict(controlled=True)},
                                  name='wraptest')
-        self.resource_names = self.default_spec['clients'].keys()
+        # TODO PY3 check if we can get rid of the list() call
```

Comment:
Here you probably don't want to, since then resource_names wouldn't be a list and could only be iterated once.